modules/tasks/src/main/java/org/apache/synapse/task/TaskManagerObserver.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.synapse.task;
+
+/**
+ * An Observer which is used to subscribe to the NTaskTaskManager
+ * initialization completion event.
+ *
+ */
+public interface TaskManagerObserver {
+	/**
+	 * Notifies all the <code>Subscribers</code> when the
+	 * link NTaskTaskManager is
+	 * successfully initialized.
+	 */
+	void update();
+}
```

Comment:
Seems intendation is wrong see https://github.com/wso2-dev/wso2-synapse/pull/163/files. 
 
Need to get rid of un-necessary white spaces.